spiketoolkit/comparison/groundtruthstudy.py

```diff
@@ -0,0 +1,386 @@
+"""
+High level tools to run many groundtruth comparison with
+many sorter on many recordings and then collect and aggregate results
+in an easy way.
+
+the all mechanism is based on an intrinsinct organisation
+into a "study_folder" with several subfolder:
+  * raw_files : contain a copy in binary format of recordings
+  * sorter_folders : contains output of sorters
+  * ground_truth : contains a copy of sorting ground  in npz format
+  * sortings: contains light copy of all sorting in npz format
+  * tables: some table in cvs format
+"""
+
+from pathlib import Path
+import os
+import json
+
+import pandas as pd
+
+import spikeextractors as se
+
+from spiketoolkit.sorters import run_sorters, loop_over_folders, collect_sorting_outputs
+from .groundtruthcomparison import compare_sorter_to_ground_truth, _perf_keys
+
+
+def setup_comparison_study(study_folder, gt_dict):
+    """
+    Based on a dict of (recordnig, sorting) create the study folder.
+    
+
+    Parameters
+    ----------
+    study_folder: str
+        The study folder.
+    
+    gt_dict : a dict of tuple (recording, sorting_gt)
+        Dict of tuple that contain recording and sorting ground truth
+    """
+    
+    study_folder = Path(study_folder)
+    assert not os.path.exists(study_folder), 'study_folder already exists'
+    
+    os.makedirs(study_folder)
+    os.makedirs(study_folder / 'raw_files')
```

Comment:
`os` and `pathlib` are well integrated in linux, but they have combatibility problems in Windows (https://github.com/cortex-lab/phy/pull/828).
It's safest to parse all Paths to str() with os calls. @samuelgarcia can you do that?

Reply:
I put this Path anywhere in SI because it was already there anywhere. I thought it was a tradition.

Reply:
I think pathlib is nice, but when you call is functions we have to parse the pathlib object as string first.

Reply:
Ah OK. I did not anderstand first.